common/on-chip-calib.cpp

```diff
@@ -468,10 +518,9 @@ namespace rs2
                         break;
                 }
 
-                if (_sub->s->supports(RS2_OPTION_EMITTER_ENABLED))
-                    _sub->s->set_option(RS2_OPTION_EMITTER_ENABLED, 0.0f);
-                if (_sub->s->supports(RS2_OPTION_THERMAL_COMPENSATION))
-                    _sub->s->set_option(RS2_OPTION_THERMAL_COMPENSATION, 0.f);
+                // TODO - When implementing UV mapping - should remove from here and handle in process_flow()
```

Comment:
What does this means?
UV mapping is implemented for D400 devices, no?

Reply:
Like discussed lets rephrased to `UV mapping based calibration`

Reply:
Rephrased to `UV mapping calibration`